build/test/httpserver_test.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+# Test the Easegress' basic functionality which is generating
+# an HTTPServer and Pipeline for testing HTTP Requests.
+# author:   benjaminwu 
+# date:     2021/0716
```

Comment:
Let's remove the author and date and put Apache 2.0 License in the head of this file.

Reply:
Updated.